assembly/pom.xml

```diff
@@ -66,9 +66,21 @@
     </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
-      <artifactId>spark-sql_${scala.binary.version}</artifactId>
+      <artifactId>xsql-sql_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-sql_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-catalyst_${scala.binary.version}</artifactId>
+      <version>${spark.version}</version>
+      <scope>provided</scope>
+    </dependency>
```

Comment:
Why there need to add catalyst dependency?

Reply:
If we don't mark spark-sql as provided, final assembly jars will contains spark-sql as well as xsql-sql module at same time which will lead to Class conflict.
![image](https://user-images.githubusercontent.com/16742851/64468152-8ed00900-d153-11e9-8f8a-20a1c8e69ae3.png)

Reply:
You means to mark `spark-sql` not `spark-catalyst`?